### ISSUE TITLE
[BUG] Enemy castbars disappear when attacking them

### ISSUE THREAD

**Issue by Trust-WoW** [balakethelock/SuperWoW#26]
**Describe the bug**
Enemy castbars disappear when I attack them. It's not even an interrupt, simply attacking them makes the castbar disappear both on nameplates and the enemy unit frame.

**To Reproduce**
1. Let the mob cast
2. Attack the mob

**Expected behavior**
Castbar should be visible even when the mob is attacked.

**Comment by balakethelock** [comment]
This has nothing to do with superwow, figure it out why it happens with the addon that does enemy castbars

**Comment by Trust-WoW** [comment]
@balakethelock 
I tested it now like 10 times and can reproduce it 100% and confirm that it is caused by SuperWoW.

If I move the dll out of the WoW folder and test ingame, everything works fine. With SuperWoW active the castbars break as described.
I tested it with ShaguPlates and also with https://github.com/TheLinuxITGuy/Turtle-Dragonflight, which allows to add the castbar to the default nameplates, both with the same result as described above.
